packages/cli/src/Generate.ts

```diff
@@ -200,9 +203,13 @@ Please run \`prisma generate\` manually.`
       )
       let hint = ''
       if (prismaClientJSGenerator) {
+        const platform = await getPlatform()
```

Comment:
I think it probably makes more sense as a local variable inside `replacePathSeperatorsIfNecessary` function, not as a parameter.

Also, you don't really need `getPlatform` from `@prisma/sdk` with all of its heavy machinery for your simple use case, just checking if `os.platform() === 'win32'` should be fine, but no harm in using `getPlatform` either of course.